appveyor.yml

```diff
@@ -0,0 +1,66 @@
+branches:
+ only:
+ - continuous-integration
+version: 2.0.{build}
+os:
+  - Visual Studio 2013
+shallow_clone: false
+skip_tags: true
+clone_folder: c:\pkmodeling
+configuration:
+  - Release
+platform:
+  - x64
+#environment:
+#  GITHUB_TOKEN:
+#    secure: mnRPU2KJBMkkUa4fo4ebLpsEG/gYpcAy4/X2ihGiwZiPGpkyvF1ItFFRSlUGubvf
+#  APPVEYOR_API_TOKEN:
+#    secure: 3+wBc5SxNqx3XjsY3EZ7afvQbLAWWcw30SPGK7q+QPA=
+before_build:
+  # Display infos
+  - cmake --version
+  # Downloads
+  - ps: |
+        $client = new-object System.Net.WebClient;
+        $client.DownloadFile("https://github.com/qiicr/zlib-dcmqi/releases/download/zlib-dcmqi-1.2.3-VS12-Win64-Release-static/zlib-dcmqi.zip", "C:\zlib-dcmqi.zip")
+        $client.DownloadFile("https://github.com/QIICR/dcmtk-dcmqi/releases/download/DCMTK-dcmqi-3.6.2_20170718-VS12-Win64-Release-v0.0.16-static/DCMTK-dcmqi.zip", "C:\DCMTK-dcmqi.zip")
+        $client.DownloadFile("https://github.com/QIICR/ITK-dcmqi/releases/download/ITK-dcmqi-VS12-Win64-Release-v0.0.26-static/ITK-dcmqi.zip", "C:\ITK-dcmqi.zip")
+  # Extract
+  - |
+    7z x C:\zlib-dcmqi.zip  -oC:\zlib-install
+    7z x C:\DCMTK-dcmqi.zip -oC:\DCMTK-install
+    7z x C:\ITK-dcmqi.zip   -oC:\ITK-install
+  # Download SlicerExecutionModel source
+  - |
+    cd c:\
+    git clone https://github.com/michaelschwier/SlicerExecutionModel.git -b for_pkmodeling SlicerExecutionModel
```

Comment:
I am curious what was the reason for this - that fork appears to be even with the upstream?

Reply:
I also created a release with the pre-build library in a zip and I couldn't do that on the original. And I thought it is better to keep the source and the build in sync, so I did it from my own fork. Alternatively we could also try to build Slicer Execution Model from scratch from the original sources when building PkModeling, since SEM is not so heavy weight as ITK and DCMTK.